.gitignore

```diff
@@ -1,3 +1,5 @@
 .pytest_cache/
 *.ipynb
 dist
+
+__pycache__
```

Comment:
you may want to put that into your global gitignore : 

https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

and or set PYTHONPYCACHEPREFIX so that these files/folder never bother you again. 

https://docs.python.org/3/using/cmdline.html#envvar-PYTHONPYCACHEPREFIX

Reply:
ooh nice thanks!